### PR TITLE
[WIP] Fix missing assets

### PR DESCRIPTION
**Pull request by victoriadotdev**
This reverts commit 9b495484672135b25a58cfa7b7fd9d3107f47f62.

### CODE REVIEW

**Comment by victoriadotdev** [comment]
Checking branch via Netlify preview, reverting the CORS changes as suggested in #135 did not seem to make a difference.

**Comment by onedrawingperday** [comment]
@victoriadotdev

Netlify deploy previews do not clear the cache:

https://stackoverflow.com/questions/50816796/how-to-create-a-netlify-deploy-preview-with-cache-clear

I suggest that you test in a browser’s private tag. If the CORS error remain in the private tab, then I suppose that the only way to test if this is fixed is by making an actual commit.

File: .gitignore [hunk 1]
```diff
@@ -1 +1,3 @@
-exampleSite/public
+exampleSite/public/
+exampleSite/resources/
+docs/css/main*
```
Comment (hanzei) [inline .gitignore]: See my comment in https://github.com/victoriadotdev/hugo-theme-introduction/pull/137

File: LICENSE [hunk 2]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2018 Vicky Lai
+   Copyright 2018 Victoria Lai
```
Comment (hanzei) [inline LICENSE]: Mind opening a separate PR for theses changes?

File: i18n/pl.toml [hunk 3]
```diff
@@ -1,25 +0,0 @@
-[404_title]
```
Comment (hanzei) [inline i18n/pl.toml]: This file should be kept and not reverted.

**Comment by hanzei** [comment]
I'm pretty sure the failing netlify deploy comes from incorrect SiteURL. Currently it is set to `https://victoria.dev/hugo-theme-introduction`. A `netlify.toml
` might fix this.

**Comment by victoriadotdev** [comment]
Okay. Sorry about the reset but this should be much better now.

**Comment by hanzei** [comment]
@victoriadotdev Please take a look at my comment above

> I'm pretty sure the failing netlify deploy comes from incorrect SiteURL. Currently it is set to https://victoria.dev/hugo-theme-introduction. A netlify.toml might fix this.

**Pull request merged**